### ISSUE TITLE
NameError: global name 'load_module' is not defined

### ISSUE THREAD

**Issue by rafifos** [arcticicestudio/snowsaw#28]
When trying to ./bootstrap, it throws this error, I've already installed the "load_module" for both python2 and python3

I'm using elementaryOS Loki (based on 16.04)

**Comment by arcticicestudio** [comment]
Thanks for your report :+1: 

Please  provide some more information so I can try to reproduce the problem:

* Are there any stacktraces or other exception messages before this one?
* Which python major version are you running the `bootstrap` script with?
* Have you tried to remove all `*.pyc` files (Run `find . -type f -name "*.pyc" -exec rm {} ;\` in your repository root) or with `PYTHONDONTWRITEBYTECODE=true ./bootstrap`?

Also please post your `bootstrap` script.

**Comment by rafifos** [comment]
TLDR; It's an environment "issue" (it isn't really an issue, it's just that some distros are using python2 as default)

So, I've managed to get it running after some digging on the code.

load_module isn't an installable module (one module with the same name already exists, and can be installed with pip), and it's only defined with Python 3.X

Ubuntu and Fedora (the ones that i've tested) comes with Python 2.7 (I guess it's 2.7), so, it won't pass the conditional checking for the python version (even if we add an alias like "alias python=python3")

The workaround I've found was to use one virtualenv to bootstrap the dotfiles, with something like "python3 -m venv .python && source .python/bin/activate && ./bootstrap"

**Comment by arcticicestudio** [comment]
Glad to see you got it working.

So again the "good" old python version problem...
It's not like Python 3 is there and stable since like 11 years, but some distros are still using this absolute outdated version as default -.-

I've never used Python 2 (Arch Linux :smile: ) and designed *snowsaw* to be compatible for Python 3 only as documented in the README, the badge at the top right below the logo banner, because in my option no one should use an outdated, unstable and unsecure/buggy version of any kind of software when there is already a new and stable (11 years !!!) major version, even the official Python documentation states to upgrade.

Anyway, I'll add a requirement to the documentation to make it clearer. Also thanks for the workaround, I'll add it to the documentation and the wiki for users that are stuck at Python 2.
